src/content/docs/en/guides/integrations-guide/markdoc.mdx

```diff
@@ -258,8 +278,7 @@ You may want to render standard Markdown elements, such as paragraphs and bolded
 
 This example renders blockquotes with a custom `Quote.astro` component:
 
-```js
-// markdoc.config.mjs
+```js title="markdoc.config.mjs"
```

Comment:
So the problem here is by highlighting "blockquote" it'll highlight even just the "blockquote" part of `notes.blockquote`, so that's not the right way to go about it here. Like the other example, I'm not convinced that we should add highlighting here because it's practically the only thing shown. 

(And again, it's `ins` not `add` for highlighting!)